modules/microshift-install-rhel-image-mode-publish-image.adoc

```diff
@@ -0,0 +1,40 @@
+// Module included in the following assemblies:
+//
+// microshift_install_bootc/microshift-install-rhel-image-mode.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-rhel-image-mode-publish-image_{context}"]
+= Publishing the bootc image to the remote registry
+
+Publish your bootc image to the remote registry so that the image can be used for running the container on another host, or for when you want to install a new operating system with the bootc image layer.
+
+.Prerequisites
+* A {op-system-base-full} 9.4 host with an active Red Hat subscription for building {microshift-short} bootc images and running containers.
+* You are logged into the {op-system} 9.4 host using the user credentials that have SUDO permissions configured.
+* You have a remote registry such as link:https:/quay.io[Red Hat quay] for storing and accessing bootc images.
+* You created the `ContainerFile` and built the image.
+
+.Procedure
+
+. Log into your remote registry by running the following command:
++
+[source,terminal]
+----
+REGISTRY_URL=quay.io
+
+$ sudo podman login "${REGISTRY_URL}" # <1>
+----
+<1> Replace _REGISTRY_URL_ with the URL for your registry.
+
+. Publish the image by running the following command:
++
+[source,terminal]
+----
+REGISTRY_IMG=myorg/mypath/"${IMAGE_NAME}" # <1> <2>
```

Comment:
Should we also set the IMAGE_NAME here? It would work if the build and push is run in the same shell, but...

```suggestion
REGISTRY_IMG=myorg/mypath/"${IMAGE_NAME}" # <1> <2>
IMAGE_NAME=microshift-4.16-bootc
```

Reply:
Let's try it.